manifests/pipelines/git-fetch-pipeline.yaml

```diff
@@ -318,8 +274,10 @@ spec:
       name: openshift-client
   - name: test-model-rest-svc
     params:
-    - name: service-name
-      value: $(tasks.sanitise-model-name.results.output-string)-$(params.model-version)
```

Comment:
This pipeline is failing now because the `test-model-rest-svc` has been updated to user `service-name` as a parameter. You can check `s3-fetch` for it